### ISSUE TITLE
Xbox360 Controller Support

### ISSUE THREAD

**Issue by SkinSpotlights** [SkinSpotlights/CreatorSuite-ReplayAPI#4]
Readd Controller Support from the Original Creator Suite

**Comment by Lite5h4dow** [comment]
by xbox 360 controllers do you mean just box 360 or all xbox controllers??

**Comment by SkinSpotlights** [comment]
> 
> 
> by xbox 360 controllers do you mean just box 360 or all xbox controllers??

Should be all xbox controllers since it uses the xinput library from Microsoft but I only have a usb xbox 360 controller to test it on, I know the xbox elite controller worked but thats the only other one I know

**Comment by Lite5h4dow** [comment]
would it be possible to have multiple mappings for things then since I have my razer orbweaver mapped up to CS and LD and would love to have my box controller as an optional camera tool

**Comment by SkinSpotlights** [comment]
> 
> 
> would it be possible to have multiple mappings for things then since I have my razer orbweaver mapped up to CS and LD and would love to have my box controller as an optional camera tool

Yes it would be purely optional, the Creator Suite prior to the Replay-API version had controller support and it was  enabled via a toggle.
